src/main.cpp

```diff
@@ -1,9 +1,50 @@
 #include <cstdlib>
 #include <iostream>
+#include <boost/program_options.hpp>
+#include <iterator>
```

Comment:
Please order standard library includes before 3rd party includes.
```suggestion
#include <iterator>

#include <boost/program_options.hpp>
```